.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License
+---
+version: 2
+updates:
+  - package-ecosystem: pip
+    directories:
+      - /clients/python
+      - /docker_tests
+      - /task_sdk
+      - /
+    schedule:
+      interval: daily
```

Comment:
I see in the documentation that it was to be "daily"
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates

Reply:
I actually don't know why they added all the quotes in the docs. But for sure the quotes are not needed.

Reply:
Well. One could say it's more "explicit".

The problem with lack of quotes is that you can easily makes mistakes there as yaml parser interpretes the values and changes the type depending of what is there:

This is boolean:

```
param: true
```

This is string:

```
param: truue
```

I personally prefer to be rather explicit and always quote string - just in case. I think avoiding quotes optimizes for writing (less characters to type), and adding quotes optimizes for reading ("you do not have to read the value to understand what type it is"). And I prefer optimizing for reading.

Reply:
But I don't mind with such configuration - I am not picky.